src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/mappers/TransitLayerMapper.java

```diff
@@ -32,15 +32,15 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Maps the TransitLayer object from the OTP Graph object. The ServiceDay hierarchy is reversed,
+ * Maps the TransitLayer object from the TransitModel object. The ServiceDay hierarchy is reversed,
```

Comment:
```suggestion
 * Maps the {@link TransitLayer} object from the {@link TransitModel} object. The ServiceDay hierarchy is reversed,
```

I prefer the linked version as this changes the docs here, when the class is renamed.

Reply:
IntelliJ has an option to extend rename operations to strings, comments etc. Where class names are clearly distinct from any plain text (via use of CamelCase) it may be sufficient to use the names. 

<img width="475" alt="Screenshot 2024-03-31 at 14 29 15" src="https://github.com/opentripplanner/OpenTripPlanner/assets/112871/9f65b3de-30d4-4915-ac79-125e0ff6da2e">

This may be a question of style and uniformity more than functionality, since an IDE can readily identify all instances of the class name. There are about 19 references in comments without the `link` notation. If we change some of them we should probably change all of them. Extending that to every class name in every comment might arguably lead to a decrease in readability or maintainability.

<img width="621" alt="Screenshot 2024-03-31 at 14 30 28" src="https://github.com/opentripplanner/OpenTripPlanner/assets/112871/3992de24-04f4-4490-b335-660564854e6c">

Of course if it is already a policy to progressively introduce `link` in any new comments then we should update these.